ui_framework/src/components/call_out/_call_out.scss

```diff
@@ -0,0 +1,57 @@
+.kuiCallOut {
+  padding: $kuiSizeM $kuiSizeL;
+  border-left: 4px solid $kuiColorPrimary;
+  background-color: rgba($kuiColorPrimary, 0.1); /* 1 */
+}
+
+// Modifier naming and colors.
+$callOutTypes: (
+  success: $kuiColorSecondary,
+  warning: $kuiColorWarning,
+  danger: $kuiColorDanger,
+);
+
+// Create button modifiders based upon the map.
+@each $name, $color in $callOutTypes {
+  .kuiCallOut--#{$name} {
+    border-color: $color;
+    background-color: rgba($color, 0.1);
+
+    .kuiCallOutHeader__icon {
+      fill: $color;
+    }
+
+    .kuiCallOutHeader__title {
+      color: $color;
+    }
+  }
+}
+
+/**
+ * 1. Align icon with first line of title text if it wraps.
+ * 2. Apply margin to all but last item in the flex.
+ */
+.kuiCallOutHeader {
+  @include kuiVerticalRhythmS;
+  @include kuiFontSizeM;
+
+  display: flex;
+  align-items: baseline; /* 1 */
+
+  > * + * {
+    margin-left: $kuiSizeS; /* 2 */
+  }
+}
+
+  /**
+   * 1. Vertically center icon with first line of title.
+   */
+  .kuiCallOutHeader__icon {
+    fill: $kuiColorPrimary;
+    transform: translateY(2px); /* 1 */
+  }
+
+  .kuiCallOutHeader__title {
+    font-weight: $kuiFontWeightMedium;
```

Comment:
Oh, I actually thought the design had a heavier weight on it. Take a look at these screenshots with the regular weight:

![image](https://user-images.githubusercontent.com/1238659/28939632-c4210e5a-7846-11e7-91b5-63dbb667c8fd.png)

![image](https://user-images.githubusercontent.com/1238659/28939634-c606c1e2-7846-11e7-848d-fe91aaba58f2.png)

It looks much lighter than the design to my eyes. To be honest, I think the heaver weight improves readability significantly since the text is the same color as the background, just darker.